GirafRest/appsettings.template.json

```diff
@@ -24,5 +24,17 @@
         "JwtKey": "<jwt-key>",
         "JwtIssuer": "<jwt-issuer>",
         "JwtExpireDays": 30
+    },
+    "IpRateLimiting": {

```

Comment:
Is this the recommended configuration ? 

When the server team is to deploy this how should they configure it ?